test/cachex/actions/get_and_update_test.exs

```diff
@@ -46,6 +48,13 @@ defmodule Cachex.Actions.GetAndUpdateTest do
         {:commit, "6"}
       end)
 
+    # ensure correct callers
+    result7 =
```

Comment:
Same here, this test is about functionality -- the `:"$callers"` passthrough is a separate test.